core/src/components/datetime/datetime-interface.ts

```diff
@@ -25,3 +25,17 @@ export interface DatetimeParts {
 export type DatetimePresentation = 'date-time' | 'time-date' | 'date' | 'time' | 'month' | 'year' | 'month-year';
 
 export type TitleSelectedDatesFormatter = (selectedDates: string[]) => string;
+
+export type DatetimeHighlightStyle =
+  | {
+      color: string;
```

Comment:
Thoughts on naming this `textColor`? A couple reasons:

1. Consistency with the `backgroundColor` key
2. I worry that devs may confuse this key with the `color` feature that can be used as a property i.e. `ion-datetime color="tertiary"`.

Reply:
I went with `color` to match the CSS property, but you make a good point re: confusion with the `color` prop on the datetime as a whole. I'll rename this 👍

Reply:
Done; playgrounds update coming soon.